doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -78,6 +78,15 @@ nRF9160
 
     * The library has been deprecated in favor of the :ref:`at_monitor_readme` library.
 
+  * :ref:`liblwm2m_carrier_readme` library:
+
+    * Added deferred event reason :c:macro:`LWM2M_CARRIER_DEFERRED_SERVICE_UNAVAILABLE`, which indicates that the LwM2M server is unavailable due to maintenance and the client will not attempt to set up a session for the next 4 hours.
```

Comment:
Same comment here about the delay for this event.

Reply:
Fixed